src/assets/Logo.tsx

```diff
@@ -1,16 +1,19 @@
 import React from 'react'
 
 type Props = {
-  color?: string
   className?: string
+  color?: string
+  monochrome?: boolean
   underlineColor?: string
 }
 
 function LogoIcon({
-  color = '#00425D',
-  underlineColor = '#00425D',
   className,
+  color = '#00425D',
+  monochrome = true,
+  underlineColor,
 }: Props): JSX.Element {
+  const defaultUnderlineColor = !monochrome ? '#FF414D' : '#00425D'
```

Comment:
```suggestion
  const defaultUnderlineColor = monochrome ? '#00425D' : '#FF414D'
```

I think negative conditions are a bit more complexe to read, can we invert this?

Reply:
Also using an enum would improve reading like : COLOR.ALMA and COLOR.MONOCHROME what do you think ?